DropwizardHelloWorld/src/main/java/com/example/helloworld/service/One.java

```diff
@@ -0,0 +1,17 @@
+package com.example.helloworld.service;
+
+import com.google.inject.BindingAnnotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Created by ccardona on 4/12/17.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.FIELD, ElementType.PARAMETER})
+@BindingAnnotation
+public @interface One {
```

Comment:
I'm curious about what are this and `@Two`annotations for? Experimenting, maybe? 🤔